### DISCUSSION TITLE
输入OpenAi密钥无法使用

### DISCUSSION THREAD

**Discussion by SerMs** [YiiGuxing/TranslationPlugin#4305] [Q&A]
请问是需要使用魔法还是怎么？
为什么我使用了密钥提示翻译超时，有什么好的解决办法没有
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/67584847/2a66e66e-d7d5-4932-bafc-5a00bbb6e26f)
![image](https://github.com/YiiGuxing/TranslationPlugin/assets/67584847/9a23ad9a-5b58-4752-99cb-2ea28a16f97c)

**Comment by YiiGuxing** [comment]
需要使用魔法的。
